src/main/java/net/twasi/obsremotejava/message/event/Event.java

```diff
@@ -0,0 +1,57 @@
+package net.twasi.obsremotejava.message.event;
+
+import net.twasi.obsremotejava.message.Message;
+
+import java.util.HashMap;
+
+public abstract class Event extends Message {
+    public static HashMap<Event.Type, Class<? extends Event>> EVENT_REGISTRY = new HashMap<>();
+
+    static {
+        Message.registerMessageType(Message.Type.Event, Event.class);
+    }
+
+    protected Type eventType;
+
```

Comment:
I see eventData has been put in the individual events, and how this could work. 👍